api-server/common/models/user.js

```diff
@@ -439,6 +439,10 @@ export default function(User) {
     )(donation).do(() =>
       this.updateAttributes({
         isDonating: true,
+        donationProviders: [
```

Comment:
What happens when you have three stripe donations, and two PayPal donations? Do we want to track repetitions? If so why?

Reply:
I am not entirely sure that we need donationProviders in the user object to be honest since the users presumably are prevented from making multiple donations, and we can pull the donation related info from the donation collection.

Reply:
Okay, in that case, you can go ahead and remove all this logic.